src/modules/party/party.service.ts

```diff
@@ -97,17 +73,44 @@ export class PartyService {
     return response;
   }
 
-  async createParty(token: string, party: CreatePartyRequestItem): Promise<{ partyIdentifier: string }> {
-    const acbsRequest = {
-      PartyAlternateIdentifier: party.alternateIdentifier,
+  async createParty(partyToCreate: CreatePartyRequestItem): Promise<CreatePartyResponse> {
+    const token = await this.getIdToken();
+    const newPartyInAcbs: AcbsCreatePartyRequest = this.buildAcbsCreatePartyRequest(partyToCreate);
+
+    return await this.acbsPartyService.createParty(newPartyInAcbs, token);
```

Comment:
same as above - we can return the service straight await `return this.acbsPartyService...`